src/interfaces/store-interfaces/state-interfaces/business-interface.ts

```diff
@@ -11,15 +12,25 @@ export interface BusinessWarningIF {
   filing: string
 }
 
+/** The Alternate Name object. */
+export interface AlternateNameIF {
+  entityType: CorpTypeCd
+  identifier: string
+  name: string
+  nameRegisteredDate: ApiDateTimeUtc
+  nameStartDate: IsoDatePacific
+  nameType: NameTypes
+}
```

Comment:
When you do Edit UI, how about moving this to the shared interfaces (and coming back and importing here and in Filings UI)?

Reply:
Sure thing! Will be doing that.

Reply:
Done! Importing and using the interface from the shared components repo now.